package.json

```diff
@@ -17,7 +17,7 @@
     "deploy:sepolia": "bash -c 'source .env && forge script Deploy --rpc-url $SEPOLIA_RPC --account $SEPOLIA_DEPLOYER_NAME --broadcast --verify --chain sepolia -vvvvv'",
     "lint:check": "yarn lint:sol && forge fmt --check",
     "lint:fix": "sort-package-json && forge fmt && yarn lint:sol --fix",
-    "lint:natspec": "npx @defi-wonderland/natspec-smells --config natspec-smells.config.js",
+    "lint:natspec": "lintspec src -f variable",
```

Comment:
how come you are using lintspec here without adding it to the package.json? I believe the dependency is missing, I don't recommend expecting people to instal it globally